src/lib/aggregations/get-bucket-script.ts

```diff
@@ -0,0 +1,15 @@
+export type BucketScriptAggregation = {
+    bucket_script: BucketScriptAggregationBody
+}
+
+export type BucketScriptAggregationBody = {
+    buckets_path: Record<string, string>
+    script: string
+}
+
+export const getBucketScriptAggregation = (bucketsPath: Record<string, string>, script: string): BucketScriptAggregation => ({
```

Comment:
maybe we switch the `script` with `bucketsPath`?

so it becomes for example (just improved readability):

```typescript
getBucketScriptAggregation('params.myVar1 / params.myVar2', {
    myVar1: 'sumAggregation',
    myVar2: 'count'
})
```

Reply:
alright